data/ecosystem/vendors.yaml

```
@@ -34,6 +34,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: Better Stack
+  distribution: false
+  nativeOTLP: true
+  url: 'https://betterstack.com/logs'
```

```suggestion
  url: https://betterstack.com/docs/logs/open-telemetry/#2-setup
```

I would prefer to have here the direct link to the documentation